app/src/androidTest/java/com/duckduckgo/app/browser/animations/TrackersRendererTest.kt

```diff
@@ -14,20 +14,16 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.privacy.dashboard.impl.animations
+package com.duckduckgo.app.browser.omnibar.animations
 
-import android.content.Context
-import androidx.test.core.app.ApplicationProvider
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.duckduckgo.privacy.dashboard.impl.R
+import androidx.test.platform.app.InstrumentationRegistry
 import org.junit.Assert.assertEquals
 import org.junit.Test
-import org.junit.runner.RunWith
+import com.duckduckgo.app.browser.R
 
-@RunWith(AndroidJUnit4::class)
 class TrackersRendererTest {
 
-    private val context = ApplicationProvider.getApplicationContext<Context>()
+    private val context = InstrumentationRegistry.getInstrumentation().targetContext
```

Comment:
I had to put these tests as androidTests. I've tried to keep them as unit test but if I set includeAndroidResources to true, all unit tests consuming application context crash. This seems to be a problem only happening in app module.